src/Contexts/section.actions.jsx

```diff
@@ -15,10 +16,7 @@ export const DELETE_SECTION = 'DELETE_SECTION'
 export const DELETING_SECTION_FAILED = 'DELETING_SECTION_FAILED'
 export const UPDATE_SECTION = 'UPDATE_SECTION'
 export const SECTION_DOCVALUE_CHANGE = 'SECTION_DOCVALUE_CHANGE'
```

Comment:
SECTION_DOCVALUE_CHANGE und updateSectionDocValue wird bisher kaum genutzt. Da könnte man drüber nachdenken, ob wir das wirklich drin lassen.

Reply:
!? das wird bei jeder änderung serlo editor states getriggert, ist vermutlich der am öftesten getriggerte type

Reply:
Das meine ich auch nicht, es wird nur einmal updateSectionDocValue verwendet in section und das einzige was dort besonders gemacht wird ist das  section.changed.add("docValue") gesetzt wird. (Wäre auch eher etwas was in die action wandern sollte.) Dann haben wir mehre reducer die einzelne Elemente der section updaten. Diese haben teilweise Logik mit drin. Aber genauso gut könnten wir mehre actions, über ein section update redurcer laufen lassen.

Das würde den code aufräumen, die Komplexität reduzieren und unerwartetes Verhalten in der Zukunft vor beugen. Performance mäßig macht es momentan wegen dem ...section kein wirklichen Unterschied. 
Das ...section würde ich im Kontext der reducer auch noch mal zur Diskussion stellen wollen. Wenn dort nur die Werte drin sind die sich wirklich ändern, ok. Aber sollte da die ganze Sektion drin sein, wäre das bzg. render Performance totaler overkill.

Mir wiederstrebt es mehrmals ähnlichen code bereit zu stellen.

Reply:
useEffect -> vs ...section --> nachlesen (ich)